Includes.ts

```diff
@@ -19,3 +20,5 @@ export function includes(needle: string, value?: any): Includes | boolean {
 	const result = new Includes(needle)
 	return value ? result.is(value) : result
 }
+
```

Comment:
Be consistent. Either an empty line before the Type.Something.add() or no empty line, don't change back and forth.